src/main/java/edu/utdallas/davisbase/representation/WhereExpression.java

```diff
@@ -0,0 +1,30 @@
+package edu.utdallas.davisbase.representation;
+
+import net.sf.jsqlparser.expression.Expression;
+
+public class WhereExpression {
+  String expression;
+  private boolean not;
+  private String column;
+  private String operator;
+  private String value;
```

Comment:
```suggestion
  private final String value;
```

Also, instead of a raw string, let's make this a literal value expression object. Otherwise, the compiler will just have to parse the string into its respective literal value type all over again.